packages/cozy-stack-client/src/__tests__/KonnectorCollection.spec.js

```diff
@@ -0,0 +1,378 @@
+jest.mock('../CozyStackClient')
+
+import CozyStackClient from '../CozyStackClient'
+import KonnectorCollection from '../KonnectorCollection'
+
+const ALL_RESPONSE_FIXTURE = {
```

Comment:
To improve lisibility maybe the fixtures could be in an external file, what do you think ?